app/config/default_parameters.yml

```diff
@@ -30,4 +30,7 @@ parameters:
     cache_redis_port: 6379
 
     # Content types included in Recommendations from YooChoose account
-    ez_recommendation.default.recommender.included_content_types: [blog, article, place]
\ No newline at end of file
+    ez_recommendation.default.recommender.included_content_types: [blog, article, place]
+
+    #Location id of the root for form uploads
+    form_builder.upload_folder.location_id: 250
```

Comment:
I think it might be this one https://github.com/ezsystems/ezstudio-demo-bundle/pull/170/files#diff-1b04a8e48802c19ae369958c12d7b317R2614